src/components/menu/menu.atoms.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, HTMLAttributes } from 'react'
+import { ElMenuItemGroup, ElMenuItemGroupTitle } from './styles'
+
+export const MenuItemGroupTitle: FC<HTMLAttributes<HTMLDivElement>> = ({ children, ...rest }) => {
+  return (
+    <li>
+      <ElMenuItemGroupTitle {...rest}>{children}</ElMenuItemGroupTitle>
+    </li>
+  )
+}
+
+export const MenuItemGroup: FC<
+  HTMLAttributes<HTMLUListElement> & {
+    title?: string
+  }
+> = ({ children, title, ...rest }) => {
+  return (
+    <ElMenuItemGroup {...rest}>
```

Comment:
**todo:** I think groups should have the `role="group"` attribute. I can see reference to this in the ARIA docs for the [menuitemgroup](https://w3c.github.io/aria/#menuitemradio) role.

Reply:
yes, since the menu could contain multiple group of checkbox, radio, button, .etc
thanks for pointing this out